package.json

```
@@ -80,7 +80,8 @@
         "@fontsource/montserrat": "^4.4.5",
         "@fontsource/noto-sans": "^4.4.5",
         "@fontsource/open-sans": "^4.5.2",
-        "@fontsource/poppins": "^4.4.5",
+        "@fontsource/opendyslexic": "^4.5.2",
+	"@fontsource/poppins": "^4.4.5",
```

pushing minor spacing fix
```suggestion
	   "@fontsource/poppins": "^4.4.5",
```